docs/reference-guides/rancher-security/selinux-rpm/about-rancher-selinux.md

```diff
@@ -16,27 +16,14 @@ The `rancher-selinux` GitHub repository is [here.](https://github.com/rancher/ra
 
 :::note Requirement:
 
-The rancher-selinux RPM was tested with CentOS 7, 8 and 9.
+The `rancher-selinux` RPM was tested on openSUSE Tumbleweed and RHEL-based distributions including Centos/RockyLinux 7, 8, and 9. 
```

Comment:
AFAIK we [no longer](https://github.com/rancher/rancher-selinux/tree/main/policy) do CentOS 7:

```suggestion
The `rancher-selinux` RPM was tested on openSUSE Tumbleweed and RHEL-based distributions including Centos/RockyLinux 8, and 9. 
```